spec/requests/errors_spec.rb

```diff
@@ -85,7 +85,11 @@
     end
 
     it "displays the correct header" do
-      expect(response.body).to match(/Access denied/)
+      expect(page).to have_css("h1", text: t("access_denied.page_title"))
```

Comment:
personally I prefer examples to test for actual content rather than than for a translations lookup, but we have a examples of both all over the app.


I think it accepts a regex?!
```suggestion
      expect(page).to have_selector("h1", text: /Access denied/)
```

Reply:
I prefer using translations because it means my test will fail if the translation is missing, plus it means I can change content without needing to update my tests.

If readability is a concern, I prefer to define well-named helper methods that can be more revealing.

This is the sort of thing I'd expect to be picked up by lint checks if the team is passionate about it one way or the other. If we're not, then 🤷 (but that's just my opinion).